docs-library/10x-FY18-Project-Results.md

```diff
@@ -0,0 +1,41 @@
+# FY18 10x Project Results
+
+## Phase One. **Investigation**
+
+**Government-wide User Testing Program** — Continued to next phase. [Final report here](https://docs.google.com/document/d/1HxCFhc27NguZwervykWLG8NMLAryWoPFQ1BiPKe-yAI/edit#heading=h.r9d3o4h63t06). Decision was made to continue this project at the $175k Discovery phase. It is clear that federal agencies face a host of barriers to conducting user research and testing. A government-wide resource helping unblock the most common barriers (such as PRA interpretation, user recruitment, a need for better design and research tools and resources, and an overall understanding of the value of research) would enable federal agencies to more effectively conduct user research and make smarter decisions when building or buying products. 
```

Comment:
I'd recommend making the project titles a level 3 heading, which does mean that the statuses (Continued to next phase etc) would need to go on the next line most likely. But that is more semantically correct than just making them bold. I would prioritize semantic HTML over formatting, since the former will better accommodate more users of assistive technology.  

Alternatively I think you could also format it as a definition list, but that sounds overly complicated.